VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -157,13 +152,7 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
         if (!value) {
             return;
         }
-        this.remoteMuted = value.pexipInfo?.isRemoteMuted;
-        this.handRaised = value.pexipInfo?.handRaised;
-
-        if (this.remoteMuted && !this.audioMuted) {
```

Comment:
is it safe to remove this if statement? I notice it hasn't been reinstated inside `updateControlBooleans`

Reply:
Ah yes, I can get rid of this now since the logic has moved into the effect localMuteParticipantOnRemoteMute$